AssetRipperGUI/MainWindow.axaml

```diff
@@ -115,7 +115,7 @@
                        TextAlignment="Center"
                        VerticalAlignment="Center" />
 
-            <Grid Grid.Row="2" RowDefinitions="50, 140, 140, 140, 140" ColumnDefinitions="*, 500, 500, *" Margin="100, 0, 100, 0">
+            <Grid Grid.Row="2" RowDefinitions="50, 140, 140, 140, 160, 140" ColumnDefinitions="*, 500, 500, *" Margin="100, 0, 100, 0">
```

Comment:
Why is the new row bigger than the others?

Reply:
I set it that high to ensure the longer text is displayed properly. With your description proposal (which is shorter) I can set it to 140.